scripts/import/uniprot/uniprot_importer.py

```diff
@@ -46,11 +46,22 @@ def get_batch(batch_url):
         yield response
         batch_url = get_next_link(response.headers)
 
-def get_database_value(database, dataItem):
+def get_database_value(database, dataItem, gene_symbol):
     if "uniProtKBCrossReferences" in dataItem and len(dataItem["uniProtKBCrossReferences"]):
         for item in dataItem["uniProtKBCrossReferences"]:
-            if item["database"] == database:
+
+            # Example: {'database': 'HGNC', 'id': 'HGNC:4764', 'properties': [{'key': 'GeneName', 'value': 'H3-3A'}]
+            if database == "HGNC" and item["properties"][0]["value"] == gene_symbol:
+                return item["id"]
+
+            # Example: {'database': 'MIM', 'id': '601058', 'properties': [{'key': 'Type', 'value': 'gene'}]}
+            elif database == "MIM" and item["properties"][0]["value"] == "gene":
```

Comment:
In these 2 if conditions, better to have a generic condition (item["database"] == database) instead of (database == "HGNC") right? This generic condition should work for both the examples

Reply:
We want to check different things for HGNC and MIM. If we only check for `item["database"] == database` we don't know which one we're accessing.